hooks/useTransactionExport.ts

```
@@ -0,0 +1,285 @@
+
+        if (permissions.granted) {
+          const safUri = await FileSystem.StorageAccessFramework.createFileAsync(
+            permissions.directoryUri,
+            filename,
+            mimeType
+          );
+          await FileSystem.StorageAccessFramework.writeAsStringAsync(safUri, content);
+          return;
+        }
+      } catch (error) {
+        console.log('SAF failed, using sharing fallback:', error);
```

Biome (1.9.4)

[error] Don't use console.log

console.log is usually a tool for debugging and you don't want to have that in production.
If it is not for debugging purpose then using console.info might be more appropriate.
Unsafe fix: Remove console.log


(lint/suspicious/noConsoleLog)